receiver/prometheusreceiver/internal/otlp_transaction.go

```diff
@@ -102,9 +104,31 @@ func (t *transaction) Append(ref storage.SeriesRef, labels labels.Labels, atMs i
 }
 
 func (t *transaction) AppendExemplar(ref storage.SeriesRef, l labels.Labels, e exemplar.Exemplar) (storage.SeriesRef, error) {
+	var exemplar ExemplarData
+	metricName := l.Get(model.MetricNameLabel)
+	familyName := normalizeMetricName(metricName)
+	sort.SliceStable(l, func(i, j int) bool {
+		return l[i].Name < l[j].Name
+	})
+	metricKey := buildMetricKey(l)
```

Comment:
Also, keep in mind that buckets and quantiles (`le` and `quantile`) labels will make this cause each bucket series to have its own metricKey.

Reply:
@dashpole Is there anything we need to address here to make it more generic?

Reply:
See my comment below about the `le` and `quantile` labels.  I'll resolve this comment